exercises/dominik_arominski/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,82 @@
+#include <fstream>
+#include <string>
+#include <map>
+#include <vector>
+#include <algorithm>
+
+std::string RemovePunctuation(std::string word) {
+  const char punctuation[] = {'.', ',', '?', '!', '"', '\'', ')', '('};
```

Comment:
This can nowadays be static constexpr, you could also take this out of the function body, and make a constant outside. In real code this should also be wrapped by anonymous namespace, to avoid any potential conflicts when linking.